### ISSUE TITLE
DESCRIBE query support

### ISSUE THREAD

**Issue by ColinMaudry** [LinkedDataFragments/Client.js#50]
Hi there :)

I am unsure whether DESCRIBE queries are supposed to be supported by the current LDF server/client versions.

DESCRIBE support seems to have been enabled by [certain commits](https://github.com/LinkedDataFragments/Client.js/search?q=describe&type=Commits). However, my testing with ldf-client@2.0.5 and ldf-server@2.2.3 seem to indicate it's not supported.

No problem with SELECT and CONSTRUCT.

From desktop to server:

```
ldf-client https://ldf.sireneld.io/sirene-sample -q "describe <https://sireneld.io/siren/006280234>"
ERROR: Query execution could not start.

The query is not yet supported
Cannot read property 'concat' of undefined
```

From server to localhost:

```
 ldf-client http://localhost:23000/sirene-sample -q "DESCRIBE <https://sireneld.io/siren/006280234>"
ERROR: Query execution could not start.

The query is not yet supported
Cannot read property 'concat' of undefined 
```

The server runs fine at https://ldf.sireneld.io/sirene-sample.

I'd like to be able to use DESCRIBE queries because that's what [Elda](http://epimorphics.github.io/elda/current/index.html) uses to get object data.

Thanks :)

**Comment by RubenVerborgh** [comment]
> I am unsure whether DESCRIBE queries are supposed to be supported

Well, it's supported, but apparently there's a bug that prevents it from working. Probably something small.

**Comment by rubensworks** [comment]
Fixed in 84cb9ec71671cfce8278b50c469d09727238f283
